server/planning/feed_parsers/onclusive.py

```diff
@@ -66,6 +66,8 @@ def parse(self, content, provider=None):
                 GUID_FIELD: guid,
                 ITEM_TYPE: CONTENT_TYPE.EVENT,
                 "state": CONTENT_STATE.INGESTED,
+                # used for event status mapping
+                "is_provisional": event.get("isProvisional", False),
```

Comment:
I'm actually wondering why not implement the logic in `set_occur_status` method and only do it in the other repo?

Reply:
You mean, instead of doing that logic in CP, we should apply that logic here?

Reply:
right

Reply:
Actually, I assumed that it was CP-specific, so I didn't apply it here.